samples/qa-drag-and-drop/.gitignore

```diff
@@ -0,0 +1,15 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
We don't need this file as it works only for the root directory of git repository.